data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented in the current version of the game."`
```

Comment:
```suggestion
				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented."`
```
The last part feels a little unnecessary and awkward.

Reply:
I included this because it appeared that @EjoThims wants to keep the phrase "the game"

Reply:
Yes, leaving the obvious lead in to the fourth wall break is necessary to avoid it for those who wish to, as well as to highlight it for the attention it's meant to capture.

Reply:
I think this can be resolved because the dialog has changed and is being decided elsewhere (in different suggestions).